src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/ImporterService.cs

```diff
@@ -505,6 +512,7 @@ private async Task InsertObservations(DbContext context, IEnumerable<Observation
                 observationsTable.Rows.Add(
                     o.Id,
                     o.SubjectId,
+                    // TODO should be able to drop GeographicLevel in future once the migration to Location is complete
```

Comment:
Ticket ref comment again

Reply:
I removed this TODO as plan to keep GeographicLevel on Observation for now but raised EES-3067 and EES-3068 and left them as comments in the appropriate place as part of EES-3011.